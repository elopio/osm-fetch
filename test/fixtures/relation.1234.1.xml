<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="OpenStreetMap server" copyright="OpenStreetMap and contributors" attribution="http://www.openstreetmap.org/copyright" license="http://opendatacommons.org/licenses/odbl/1-0/">
  <relation id="1234" changeset="13368155" timestamp="2012-10-05T03:00:00Z" version="1" visible="true" user="Bleuet Mapper" uid="762332">
    <member type="way" ref="4321" role="outer"/>
    <tag k="natural" v="wetland"/>
    <tag k="source" v="NRCan-CanVec-10.0"/>
  </relation>
</osm>
