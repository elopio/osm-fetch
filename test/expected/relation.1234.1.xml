<?xml version='1.0' encoding='UTF-8'?><osm version="0.6" generator="osm-fetch"><node id="1234" changeset="13368155" timestamp="2012-10-05T03:00:00Z" version="3" visible="true" user="Bleuet Mapper" uid="762332" lat="48.9198889" lon="-71.6481385"/><way id="4321" changeset="13368155" timestamp="2012-10-05T02:00:00Z" version="1" visible="true" user="Bleuet Mapper" uid="762332"><nd ref="1234"/></way><relation id="1234" changeset="13368155" timestamp="2012-10-05T03:00:00Z" version="1" visible="true" user="Bleuet Mapper" uid="762332"><member type="way" ref="4321" role="outer"/><tag k="natural" v="wetland"/><tag k="source" v="NRCan-CanVec-10.0"/></relation></osm>
